### ISSUE TITLE
Support of other DBs

### ISSUE THREAD

**Issue by nibynool** [Nevon/serverless-pg-migrations#3]
From what I can tell (and please forgive me if I'm wrong), but this plugin looks as though it should support any DB that can be used with Sequelize, is that correct?

If it does would it be appropriate to remove the dependency on `pg` and allow users to select their preferred database?

**Comment by Nevon** [comment]
Yes, I believe that you are correct. I suspect that if you supply a connection string for a different DB, Sequelize will try to use whatever support module is necessary for that DB.

If so, we could move `pg` to be a peer dependency instead (because we want to make sure that people don't use `pg` >= 7, as it doesn't work with Sequelize currently). Feel free to give it a try and submit a PR if it works.
